gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -632,11 +629,7 @@ class {{ service.async_client_name }}:
 
         # Wrap the RPC method; this adds retry and timeout information,
         # and friendly error handling.
-        rpc = gapic_v1.method_async.wrap_method(
-            self._client._transport.get_iam_policy,
-            default_timeout=None,
-            client_info=DEFAULT_CLIENT_INFO,
-        )
+        rpc = self.transport._wrapped_methods[self._client._transport.get_iam_policy]
```

Comment:
I may have updated all the instances just because it makes sense to use `wraped_methods` everywhere. However, the methods defined in async client aren't generated / used. There's a linked PR which mentions that those methods should be cleaned up. We can follow up on it. Thanks for filing the issue!